[2021-10-02 04:10:36,745] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_C 2021-10-02T03:59:45.116659+00:00 [queued]>
[2021-10-02 04:10:37,125] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_C 2021-10-02T03:59:45.116659+00:00 [queued]>
[2021-10-02 04:10:37,135] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 04:10:37,137] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-02 04:10:37,140] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 04:10:37,225] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): Training_Model_C> on 2021-10-02T03:59:45.116659+00:00
[2021-10-02 04:10:37,278] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Training_Model_C', '2021-10-02T03:59:45.116659+00:00', '--job-id', '7', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpqdd8vs9p', '--error-file', '/tmp/tmprndjix6z']
[2021-10-02 04:10:37,280] {standard_task_runner.py:77} INFO - Job 7: Subtask Training_Model_C
[2021-10-02 04:10:37,274] {standard_task_runner.py:52} INFO - Started process 17332 to run task
[2021-10-02 04:10:37,493] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Training_Model_C 2021-10-02T03:59:45.116659+00:00 [running]> on host d6b4131456ad
[2021-10-02 04:10:37,881] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Training_Model_C
AIRFLOW_CTX_EXECUTION_DATE=2021-10-02T03:59:45.116659+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-02T03:59:45.116659+00:00
[2021-10-02 04:10:37,890] {python.py:151} INFO - Done. Returned value was: 6
[2021-10-02 04:10:37,966] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Training_Model_C, execution_date=20211002T035945, start_date=20211002T041036, end_date=20211002T041037
[2021-10-02 04:10:38,091] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-02 04:10:38,330] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
